server/main/src/main/resources/production.conf

```diff
@@ -0,0 +1,38 @@
+include "main.conf"
+
+akka {
+  loglevel = DEBUG
+
+  remote {
+    netty.tcp {
+      hostname = ${?APP_ADDR}
+      port = 2552
+    }
+  }
+
+  cluster {
+    seed-nodes = [ ${?SEED_NODES} ]
+    jvm-ports = [ ${akka.remote.netty.tcp.port} ]
+    retry = 10s
+    min-nr-of-members = 2
+  }
+
+  persistence {
+    journal.plugin = "cassandra-journal"
+    snapshot-store.plugin = "cassandra-snapshot-store"
+  }
+}
+
+cassandra-journal {
+  contact-points = [ ${?JOURNAL} ]
+}
+
+cassandra-snapshot-store {
+  contact-points = [ ${?SNAPSHOT} ]
+}
+
+kafka {
```

Comment:
I think @carlpulley you are correct in saying that only the `metadata.broker.list` value is overridden in `production.conf`. Perhaps for better readability you might consider adding an inline comment to explain. I must add that it would mean "idiot proofing" the code! But, sometimes I am the idiot reading the code, so you might consider me biased :smile: 
You might choose to ignore my comment as well :laughing: